src/components/ComboCount/countShake.css

```diff
@@ -0,0 +1,31 @@
+.comboShake {
+    animation: shake 0.41s cubic-bezier(.36,.07,.19,.97) both;
+    transform: translate3d(0, 0, 0);
+    backface-visibility: hidden;
+    perspective: 1000px;
+  }
+
+.comboShakeEnd {
+animation: shake 0.41s cubic-bezier(.36,.07,.19,.97) both;
+transform: translate3d(0, 0, 0);
+backface-visibility: hidden;
+perspective: 1000px;
+}
```

Comment:
One last thing you have to do: `npm install`
I think prettier was supposed to fix the indentation here

Reply:
Ran the npm install